src/libs/ReportUtils.ts

```diff
@@ -1415,6 +1412,23 @@ function canCreateTaskInReport(report: OnyxEntry<Report>): boolean {
     return true;
 }
 
+/**
+ * For all intents and purposes a report that has no notificationPreference at all should be considered "hidden".
+ * We will remove the 'hidden' field entirely once the backend changes for https://github.com/Expensify/Expensify/issues/450891 are done.
+ */
+function isHiddenParticipant(notificationPreference: string | null | undefined): boolean;
+function isHiddenParticipant(report: OnyxEntry<Report>): boolean;
+function isHiddenParticipant(reportOrPreference: OnyxEntry<Report> | string | null | undefined): boolean {
```

Comment:
I feel like this is a misnomer, to me it looks like this function would be about a participant being hidden, and not that they have a "hidden" notification preference. Wdyt?

Reply:
Good point, I think that `isReportHidden` or `isHiddenForCurrentUser` or something like that might be clearer

Reply:
I can see that. I think the confusion is that having this notification preference actually means both: 

- hidden from other participants
- hidden from the LHN

🤷‍♂️

Reply:
`isReportHidden` seems too broad since it's not the report but the participant that is actually "hidden".

`currentUserHasHiddenNotificationPreference()` would be the most explicit maybe.

Reply:
Gonna go with `isHiddenForCurrentUser()` though I think it's fine.

Reply:
I think `hasHiddenNotificationPreference` would work as well? we can deduce who by finding out whose preferences we're passing to the function.